li-utils/src/main/pegasus/com/linkedin/common/MLFeatureDataType.pdl

```diff
@@ -0,0 +1,47 @@
+namespace com.linkedin.common
+
+/**
+ * MLFeature Data Type
+ */
+enum MLFeatureDataType {
```

Comment:
This was mentioned in the RFC. Looks like there are some specific data types for ML features https://towardsdatascience.com/7-data-types-a-better-way-to-think-about-data-types-for-machine-learning-939fae99a689
Perhaps we can expand the current enum to include these? Not all might be mutually exclusive.

Reply:
Found datatypes like `NUMBER` and `COUNT` to be very close but not quite. As `COUNT` does not contain negative numbers as well as long or double number types. In that case those numbers would be part of `ORDINAL`?
With the above Assumption removed `NUMBER`.
`BOOLEAN` would fit in the newly defined `BINARY` type.
`DATETIME` would fit in the newly defined `TIME` data type?
I didn't see any equivalent for `SET`, `MAP` or `SEQUENCE` so left those types as is.

Let me know your thoughts.

Reply:
Sure.